chromium_src/chrome/browser/process_singleton_win.cc

```diff
@@ -250,7 +250,8 @@ void ProcessSingleton::OnBrowserReady() {}
 // isn't one, create a message window with its title set to the profile
 // directory path.
 bool ProcessSingleton::Create() {
-  static const wchar_t kMutexName[] = L"Local\\AtomProcessSingletonStartup!";
+  static const wchar_t kMutexName[] =
+      L"Local\\ElectronProcessSingletonStartup!";
```

Comment:
I'm not sure what the implications of changing this are... cc @MarshallOfSound maybe?

Reply:
reverted this for now so as not to block.

Reply:
@nornagon Can we add a comment here to not change this without prior planning, if we change this we'll need to support both names for a while as we want the singleton to work across app updates

Reply:
I just reverted the change, we can deal with it separately.